core/domain/question_services.py

```diff
@@ -147,15 +156,15 @@ def apply_change_list(question_id, change_list):
     Returns:
       Question. The resulting question domain object.
     """
-    question = get_question_by_id(question_id)
+    question = get_question_by_id(question_id, strict=False)
```

Comment:
Why strict=False? In general if you don't expect an error then error noisily. If you use strict=False then you must deal explicitly with the case question = None.

Reply:
Done, missed these calls.